R/expect-setequal.R

```diff
@@ -117,3 +121,53 @@ check_names_ok <- function(x, label) {
     stop("All elements in `", label, "` must be named")
   }
 }
+
+#' @export
+#' @rdname expect_setequal
+expect_contains <- function(object, expected) {
+  act <- quasi_label(enquo(object), arg = "object")
+  exp <- quasi_label(enquo(expected), arg = "expected")
+
+  if (!is_vector(act$val) || !is_vector(exp$val)) {
+    abort("`object` and `expected` must both be vectors")
+  }
+
+  exp_miss <- !exp$val %in% act$val
+
+  if (any(exp_miss)) {
+    fail(paste0(
+      act$lab, " (`actual`) doesn't contain all the values in ", exp$lab, " (`expected)`.\n",
+      paste0("* Missing from `actual`: ",  values(exp$val[exp_miss]), "\n"),
+      paste0("* Present in `actual`:   ",  values(act$val), "\n")
+    ))
+  } else {
+    succeed()
+  }
+
+  invisible(act$val)
+}
+
+#' @export
+#' @rdname expect_setequal
+expect_in <- function(object, expected) {
+  act <- quasi_label(enquo(object), arg = "object")
+  exp <- quasi_label(enquo(expected), arg = "expected")
+
+  if (!is_vector(act$val) || !is_vector(exp$val)) {
+    abort("`object` and `expected` must both be vectors")
+  }
+
+  act_miss <- !act$val %in% exp$val
+
+  if (any(act_miss)) {
+    fail(paste0(
+      "All values of ", act$lab, " (`actual`) aren't contained in ", exp$lab, " (`expected`).\n",
```

Comment:
Alternative suggestion? I like that `act$lab` comes first, at least. 

```suggestion
      act$lab, " (`actual`) isn't fully contained within ", exp$lab, " (`expected`).\n",
```

`fully` is optional but you could also do this in `expect_contains()` to make fairly symmetric messages:

```
act$lab, " (`actual`) doesn't fully contain ", exp$lab, " (`expected)`.\n",
```

I'm not super picky about this though so if you like what you have that is fine too